VCFv4.3.tex

```diff
@@ -326,132 +327,147 @@ \subsubsection{Fixed fields}
-order (colon-separated FORMAT ids matching the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed). This is followed by one data block per
-sample, with the colon-separated data corresponding to the types
-specified in the format. The first sub-field must always be the genotype (GT)
-if it is present.  There are no required sub-fields.
+If genotype information is present, then the same types of data must be present for all samples. First a FORMAT field is given specifying the data types and order (colon-separated FORMAT ids matching the regular expression \texttt{\^{}[A-Za-z\_][0-9A-Za-z\_.]*\$}, duplicate fields are not allowed). This is followed by one data block per sample, with the colon-separated data corresponding to the types specified in the format. The first sub-field must always be the genotype (GT) if it is present. There are no required sub-fields. Additional Genotype fields can be defined in the meta-information, however, software support for such fields is not guaranteed.
+
+If any of the fields is missing, it is replaced with the missing value. For example if the FORMAT is GT:GQ:DP:HQ then $0\mid0:.:23:23,34$ indicates that GQ is missing. Trailing fields can be dropped, with the exception of the GT field, which should always be present if specified in the FORMAT field.
+
+See table~\ref{table:reserved-genotypes} for a list of common, reserved keywords that are standards across the community, and the following points for detailed explanations about the most complex among them. See also Section~\ref{sv-format-keys} for a list of genotype keys reserved for structural variants.
+
+\begin{table}[htbp]
+  \centering
+  \begin{tabularx}{\textwidth}{ | l | l | l | X | }
+      Field		& Number	& Type		& Description \\ \hline
+      AD		& R		& Integer	& Per-sample read depth for each allele \\
+      ADF		& R		& Integer	& Per-sample read depth for each allele on the forward strand \\
+      ADR		& R		& Integer	& Per-sample read depth for each allele on the reverse strand \\
+      DP		& 1		& Integer	& Per-sample read depth \\
+      EC		& A		& Integer	& Expected alternate allele counts for each alternate allele, in the same order as listed in the ALT field (typically used in association analyses) \\
+      FT		& 1		& String	& Filter indicating if this genotype was ``called''. More details in Subsection~\ref{para:genotype-FT} \\
+      GL		& G		& Float		& $log_{10}$-scaled likelihoods for all possible genotypes given the set of alleles defined in the REF and ALT fields. More details in Subsection~\ref{para:genotype-GL} \\
+      GP		& G		& Float		& Genotype posterior probabilities in the range 0 to 1 using the same ordering as the GL field (could be used to store imputed genotype probabilities) \\
+      GQ		& 1		& Integer	& Conditional genotype quality, encoded as a phred quality $-10log_{10}$ p(genotype call is wrong, conditioned on the site's being variant) \\
```

Comment:
This made me curious: Does any caller actually calculate this as written? GATK simply puts the difference of the two best PLs...which is more like the odds-ratio of the best two genotypes, not the P(error).